### PR TITLE
Adding a stub of a file for lelar66-recommendations

### LINKED ISSUE #16: lelar66's recommendation for things to do/eat
### \-  things to do
- KidSpace in Pasadena
- Huntington Library children's garden
### -eat
- soup plantation

### PR DESCRIPTION
**Pull request by lelar66**
:+1: 
:warning: Work IN PROGRESS
closes #16

### CODE REVIEW

File: lelar66-recommends.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+s is lelar66-recommends.md file
+it has recommendations for what to do in pasadena with a 4-yr old boy
+places to eat and things to do
+## Recommendations for Pasadena,CA
+### Places to Eat
```
Comment (brianamarie) [inline lelar66-recommends.md:5]: I especially love BBQ. Any recommendations?

File: lelar66-recommends.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+### Things to Do
```
Comment (brianamarie) [inline lelar66-recommends.md:8]: I'd love to see some live music, or go someplace fun to watch the Cubs/Dodgers game. 🎹 ⚾️ Is there anywhere good for those things?

**Review by brianamarie — COMMENTED** [review]
Adding a few comments about the types of places I'd like to go. 😄

**Review by brianamarie — DISMISSED** [review]
I request changes to this so I follow directions. 😉

File: lelar66-recommends.md (hunk 1 above)
Comment (lelar66) [reply lelar66-recommends.md:5]: if you have never tried Korean BBQ it is great!

File: lelar66-recommends.md (hunk 2 above)
Comment (lelar66) [reply lelar66-recommends.md:8]: check dodgers schedule they might be in LA.
There are some jazz clubs in Pasadena... Red White and Blue Jazz

**Review by brianamarie — APPROVED** [review]
👍 Awesome, looks good to merge! 🚢 :shipit:

**Pull request merged**